src/main/java/com/epam/spring/homework3/service/repository/ServiceRepository.java

```diff
@@ -0,0 +1,15 @@
+package com.epam.spring.homework3.service.repository;
+
+import com.epam.spring.homework3.service.model.Service;
+
+import java.util.List;
+
+public interface ServiceRepository {
+    Service insertService(Service service);
+
+    Service selectService(int id);
```

Comment:
I would recommend instead of select... rename to get..., it has more readability. Also check this in other classes